scripts/release.sh

```diff
@@ -49,11 +49,11 @@ help() {
 
 main() {
   VERSION=$1
-  if [[ ! "${VERSION}" =~ [0-9]+.[0-9]+.[0-9]+ ]]; then
+  if [[ ! "${VERSION}" =~ ^v?[0-9]+.[0-9]+.[0-9]+ ]]; then
     log_error "Expected 'version' param of the form '<major-version>.<minor-version>.<patch-version>' but got '${VERSION}'"
     exit 1
   fi
-  RELEASE_VERSION="v${VERSION}"
+  RELEASE_VERSION="v${VERSION#v}"
```

Comment:
I think we can merge this enhancement separately.

Reply:
I'll repurpose this pull request only to have this enhancement. And will open up another one to update the release documentation.

Reply:
Done.